admin/uploads.rb

```diff
@@ -88,25 +91,45 @@
 	  end
     column "" do |upload|
       result = ""
-      result += link_to(t(:view), admin_upload_path(upload), class: "member_link edit_link view", title: I18n.t("active_admin.uploads.title1"))
-      result += link_to(t(:edit), edit_admin_upload_path(upload), class: "member_link edit_link edit", title: I18n.t("active_admin.uploads.title2"))
-      result += link_to(t(:delete), admin_upload_path(upload), method: :DELETE, "data-confirm" => t("delete_article", scope: [:goldencobra, :flash_notice]), class: "member_link delete_link delete", title: I18n.t("active_admin.uploads.title3"))
+      result += link_to(
+        t(:view),
+        admin_upload_path(upload),
+        class: "member_link edit_link view",
+          title: I18n.t("active_admin.uploads.title1")
+      )
+      result += link_to(
+        t(:edit),
+        edit_admin_upload_path(upload),
+        class: "member_link edit_link edit",
+          title: I18n.t("active_admin.uploads.title2")
+      )
+      result += link_to(
+        t(:delete),
+        admin_upload_path(upload),
+        method: :DELETE,
+        "data-confirm" => t("delete_article", scope: [:goldencobra, :flash_notice]),
+        class: "member_link delete_link delete",
+        title: I18n.t("active_admin.uploads.title3")
+      )
       raw(result)
     end
   end
 
   show do
+    protocol = Goldencobra::Setting.for_key("goldencobra.use_ssl") == "true" ? "https" : "http"
```

Comment:
Searching across Goldencobra for the occurences of `Goldencobra::Setting.for_key("goldencobra.use_ssl")` reveals at least 5 hits for the current master branch. Adding a 6th here doesn't help the cause of bettering Goldencobra.

We should strive for a solution where building an url is encapsulated somewhere else. It always comes down to building a url from the settings of `Goldencobra::Setting.for_key("goldencobra.use_ssl")` and `Goldencobra::Setting.for_key("goldencobra.url")`. I don't like particularly, that you build the url anew for every row in the table.